samples/app-installation-using-qr-code/nodejs/.vscode/tasks.json

```diff
@@ -28,8 +28,7 @@
                     "portOccupancy" // Validate available ports to ensure those debug ones are not occupied.
                 ],
                 "portOccupancy": [
-                    3978, // app service port
-                    9239 // app inspector port for Node.js debugger
```

Comment:
second port not required ?

Reply:
Yes only server is running 3978, there is no client in this sample